tests/system/system_tests.cfg

```diff
@@ -1,395 +1,7 @@
-[System-base]
-test_description = Short simulation to ensure system tests configuration works
-global_parameter_file = global_param.ROSEM.system.txt
+[System-check-domainfile-and-nans] 
+driver = image
+test_description = check that coordinates and mask in domain and output files match 
+global_parameter_file = global_param.image.STEHE.feb.txt 
```

Comment:
Hmm I think we want to commit this global parameter template file as well? (also a minor point - I personally think no need to put "feb" in the file name, since this global file should work both for water balance and energy balance I suppose?)